File: problem 16/Option 01/BoxTest.cs
class Box
{
    private float height;
    private float length;
    private float width;

    public BoxTest(float hg, float ln, float wd){
        height = hg;
        length = ln;
        width = wd;
    }

    public float CalculateVolume(){
        return height*width*height;
    }
}

public class BoxTest 
{
  static void Main(string[] args)
  {
    Box myBox = new Box(7, 8, 3);
    Console.WriteLine(myBox.calculateVolume());
  }
}

File: problem 16/Option 01/BoxTest.py
#!python
class Box:
    length: float
    height: float
    width: float

    def __init__(self, h: float, l: float, w: float):
        self.length = l
        self.height = h
        self.width = w
    
    def calculateVolume(self):
        return (self.length * self.height * self.width)
        

if __name__ == '__main__':
    Boxed = Box(7, 8, 3)
    print("The volume of a box is :", Boxed.calculateVolume())

# on cmd navigated DIR just write " python -u BoxTest.py " if you have python3 run app on your machine.
